### ISSUE TITLE
Add OxyPlot.Export repository

### ISSUE THREAD

**Issue by tibel** [oxyplot/oxyplot#773]
I think oxyplot project/repo should concentrate on plot rendering on screen only.
So it would make sense to extract the exporting/reporting funcionalities to a separate repository.

I suggest to create a new repo oxyplot-export and move following out of the oxyplot repo:
- OxyPlot.OpenXml
- OxyPlot.Pdf
- OxyPlot.Xps
- from OxyPlot.Core
  - Pdf
  - Reporting
  - Svg

**Comment by objorke** [comment]
OpenXml, Pdf and Xps will add different dependencies, wouldn't it be best to keep these in separate packages.
I agree Reporting could be moved out of OxyPlot.Core - maybe to an OxyPlot.Contrib project? #771 
The core Pdf and Svg export should be no problem to move to a new OxyPlot.Export, but is it really worth it (keep it simple..)

**Comment by tibel** [comment]
I don't want to put everything into one nuget package but to put it together in one repository.
There will be multiple nuget packages with the focus on exporting.
They should only depend on OxyPlot.Core package.

I think this separation of concerns will make it simple.

**Comment by objorke** [comment]
I agree on moving all the reporting functionality to a separate repository (oxyplot-reporting). The reporting model could even be in a separate project (not OxyPlot).

But I think the PDF and SVG export in the Core library could remain. These should be part of the core functionality in my opinion.

**Comment by objorke** [comment]
I suggest to move the code related to the report model to a new project OxyPlot.Reporting in the main repository as the first step.
